dist/scss/wds-components/_avatar-stack.scss

```diff
@@ -24,4 +12,16 @@
 		transform: translate(-40px);
 		width: auto;
 	}
+
+	.wds-avatar {
+		color: $wds-color-light-c4;
+	}
+
+	@for $i from 1 to 6 {
```

Comment:
this could be moved to .wds-avatar
`&:nth-child(#{$i})`

Reply:
@hakubo I would say not to do this, in mobile-wiki we don't use avatar stack at all, so why would we load this styles?